Buildings/Controls/OBC/OutdoorLights/DaylightControlled.mo

```diff
@@ -12,10 +12,19 @@ protected
   final lat=lat,
   final lon=lon,
   final timZon=timZon)
-   annotation (Placement(transformation(extent={{-20,-10},{0,10}})));
+   annotation (Placement(transformation(extent={{-80,-10},{-60,10}})));
+  CDL.Logical.Not not1
```

Comment:
Every instance needs a short comment. Please update the file accordingly.

Reply:
Sorry I forgot again...Added already.